packages/fxa-settings/config/env.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
```

Comment:
Will need to add license header to the files.

```
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Reply:
hmm this is one of the many files that the eject script copied over, so I have no idea whether we should or need to put the Moz license header on them.  They are actually from react-scripts after all.

Reply:
Should we instead add a header to indicate that these are from react-scripts?

Reply:
> Should we instead add a header to indicate that these are from react-scripts?

Sounds good.  I'll add a comment but not a license header.